### DISCUSSION TITLE
FittedBox - Autoscaling

### DISCUSSION THREAD

**Discussion by Dejv311** [flet-dev/flet#4092] [Ideas]
I am trying to figure out how to **scale text** to screen width. (i.e. text should cover the entire width of the screen - the size of the text should be set accordingly). I found this flutter solution: **FittedBox**.

https://api.flutter.dev/flutter/widgets/FittedBox-class.html

Looking further into FittedBox I have to say it is truly amazing and should be **added into Flet**. What do you guys think?

**Comment by ndonkoHenri** [comment]
I created an issue https://github.com/flet-dev/flet/issues/4109
